src/stdafx.h

```diff
@@ -420,19 +420,30 @@ assert_compile(SIZE_MAX >= UINT32_MAX);
 	#define CloseConnection OTTD_CloseConnection
 #endif /* __APPLE__ */
 
+#ifdef __GNUC__
+#define likely(x)       __builtin_expect(!!(x), 1)
+#define unlikely(x)     __builtin_expect(!!(x), 0)
+#else
+#define likely(x)       (x)
+#define unlikely(x)     (x)
```

Comment:
These `#define`s should be indented. However, the commit checker refused to let me do so.

Reply:
You need to keep the `#` in the first column un-indented, and then indent the keyword (with tabs), like this:
```
#ifdef A
#	define likely(x)
#else
```

Reply:
(this is according to "The '#' of preprocessor instructions goes into the first column of a line. Indenting is done after the '#' (using tabs again)." from https://wiki.openttd.org/Coding_style#Other_important_rules)

Reply:
> You need to keep the `#` in the first column un-indented, and then indent the keyword (with tabs), like this:
> 
> ```
> #ifdef A
> #	define likely(x)
> #else
> ```

Thanks for the feedback. I can certainly do that; and had actually thought about doing so; but that would then be inconsistent with the rest of the file (and possibly countless others like it).

Reply:
Yeah, codestyle was changed, but it's only being required for "new" code. At some point someone will probably go in and fix it all...

Reply:
> Yeah, codestyle was changed, but it's only being required for "new" code. At some point someone will probably go in and fix it all...

Be careful what you ask for, @LordAro. I have a `sed`, and I know how to use it B-}